dashboard/src/components/StageProgressBar.tsx

```
@@ -0,0 +1,221 @@
+  Box,
+  Typography,
+  Tooltip,
+  Step,
+  StepLabel,
+  Stepper,
+  StepContent,
+  Chip,
+} from '@mui/material';
+import {
+  CheckCircle,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)